sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -986,6 +986,7 @@ private[hive] object HiveClientImpl {
     tpart.setTableName(ht.getTableName)
     tpart.setValues(partValues.asJava)
     tpart.setSd(storageDesc)
+    tpart.setCreateTime((System.currentTimeMillis() / 1000).toInt)
```

Comment:
@gatorsmile sorry for my misunderstanding of your point. `toHivePartition` and `fromHivePartition` should be symmetric, and I change my implementation. could you please review it again?